packages/types/src/customer/common.ts

```diff
@@ -1,6 +1,25 @@
 import { AddressDTO } from "../address"
 
-export interface CustomerDTO {
+export type CustomerDTO = {
```

Comment:
question: We generally use interfaces for DTOs. Is this an intentional change?

Reply:
Nope not really - just noticed the legacy_CartDTO was a type so think that threw me off;

is there any reason for interface vs. type

Reply:
Interfaces are extendable not the types so it make it easier for building other interfaces in some cases. But also easier if a user want to extend interfaces as most user doesn't know how to play with types i would say